x/datadeal/README.md

```diff
@@ -0,0 +1,52 @@
+## Data Deal Commands
+
+### Create a Deal
+```shell
+CREATOR=$(panacead keys show {your deal creator} -a)
+
+panacead tx datadeal create-deal --deal-file {your deal json file path} --from $CREATOR --chain-id {your chain ID}
+```
+Deal JSON File Example
+```json
+{
+  "data_schema": [
+    "https://www.json.ld"
+  ],
+  "budget": "10000000umed",
+  "max_num_data": 10000,
+  "trusted_data_validator": [
+    "...."
+  ]
+}
+```
+
+### Sell Data
+```shell
+SELLER=$(panacead keys show {your seller} -a)
+
+panacead tx datadeal sell-data --data-verification-certificate-file {your data cert json file path} --from $SELLER --chain-id {your chain ID}
```

Comment:
How about we rename `--data-verification-certificate-file` to `--data-certificate-file`?

Reply:
yeah. or, `--data-cert-file`.

Reply:
The `--data-cert-file` seems to be good!